apps/native/screens/PropertyManagementScreen.tsx

```diff
@@ -112,7 +112,7 @@ const CondoProfileScreen = () => {
       </ScrollView>
       <View id="addPropertyBtnView" style={styles.addPropertyBtn}>
         <Button
-          text="Add New Property"
+          text="Add New Property" 
```

Comment:
Change this to register new property since adding is like creating a new one in the system, but this should be just associating to userId.